community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -197,7 +195,7 @@ public GraphDatabaseFacade()
     /**
      * Create a new Core API facade, backed by the given SPI.
      */
-    public void init( SPI spi, Config config )
+    public void initSPI( SPI spi, Config config )
```

Comment:
I tried this, but it failed a lot of tests that do not need the TransactionalContextFactory. It seems like here is work to do, but I would rather not spend more time on this is the scope of this PR.